### ISSUE TITLE
1.0.0-preview2-003093 broken

### ISSUE THREAD

**Issue by andschwa** [dotnet/cli#3570]
## Steps to reproduce

Install [1.0.0-preview2-003093](https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/1.0.0-preview2-003093/dotnet-dev-ubuntu-x64.1.0.0-preview2-003093.tar.gz) on Ubuntu 14.04, run any command (say, `dotnet --info`).

## Expected  behavior

`dotnet` to work

## Actual behavior

```sh
dotnet --version
Failed to resolve library symbol hostfxr_main, error: dotnet: undefined symbol: hostfxr_main
Segmentation fault (core dumped)
```

## Environment data
`dotnet --info` output:

```
Failed to resolve library symbol hostfxr_main, error: dotnet: undefined symbol: hostfxr_main
Segmentation fault (core dumped)
```

Even stranger is that it seems to not be the build that I installed:

```
$ COREHOST_TRACE=1 dotnet -h
Tracing enabled
--- Invoked dotnet [version: 1.0.1, commit hash: d550e242a646770ed0652b40027221b9b2faf53d] main = {
dotnet
-h
}
Failed to resolve library symbol hostfxr_main, error: dotnet: undefined symbol: hostfxr_main
Segmentation fault (core dumped)
```

**Comment by brthor** [comment]
We are aware of this issue. Fix incoming.

**Comment by andschwa** [comment]
Thanks @brthor!

**Comment by brthor** [comment]
LKG version is `1.0.0-preview2-003067`

**Comment by andschwa** [comment]
Perfect, that just saved me some hunting.

**Comment by brthor** [comment]
This issue only exists for tar.gz/zips, installers should be unaffected.

**Comment by andschwa** [comment]
`dotnet-install.(sh,ps1) -c preview -v 1.0.0-preview2-003067` is the safest/easiest way to get LKG 1.0.0 version, yeah?

**Comment by brthor** [comment]
#3572 patching stage0 underway then this will go in

**Comment by brthor** [comment]
use `1.0.0-preview2-003096`

**Comment by brthor** [comment]
@schellap This is a bad error when hostfxr can't be found though

**Comment by Choleriker** [comment]
I have had this error on Ubuntu 16.04 and 14.04.

I have had the problem when installing like this:

```
sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
sudo apt-get update
sudo apt-get install dotnet-dev-1.0.0-preview1-002702
```

But when I download and installing the dotnetcore DEB package from here
https://github.com/dotnet/cli#installers-and-binaries

All is working well!

**Comment by draptik** [comment]
thanks, @brthor using version `1.0.0-preview2-003096` works!

```
sudo apt-get install dotnet-dev-1.0.0-preview2-003096
dotnet --version
1.0.0-preview2-003096
```

**Comment by brthor** [comment]
Thanks for the confirmation @Choleriker @draptik :+1:

**Comment by lgreenlee** [comment]
Thanks! This resolved my issue after upgrading to 16.04. 

sudo apt-get install dotnet-dev-1.0.0-preview2-003096

works for me too
